doc/proposals/dualstack.md

```diff
@@ -0,0 +1,280 @@
+# Whereabouts support for Ipv4/IPv6 DualStack
+
+# Table of contents
+
+- [Introduction](#introduction)
+  - [Goal](#goal)
+- [Design](#design)
+  - [Changes in IPAM Config](#configChanges)
+  - [Changes in Modules](#moduleChanges)
+- [Alternative Design](#alternative)
+- [Summary](#summary)
+
+<hr>
+
+## Introduction <a name="introduction"></a>
```

Comment:
Isn't this `<a name="introduction"></a> ` redundant in markdown ? AFAIU, there's an implicit anchor after the lowercase, dash separated, name of the heading. In this case - introduction.